packages/types/src/clerk.ts

```diff
@@ -1035,6 +1043,15 @@ export type UserButtonProps = UserButtonProfileMode & {
    * Controls the default state of the UserButton
    */
   defaultOpen?: boolean;
+
+  /**
+   * If true the `<UserButton />` will only render the popover.
+   * Enables developers to implement a custom dialog.
+   * @experimental This API is experimental and may change at any moment.
+   * @default undefined
+   */
+  __experimental_asStandalone?: boolean;
```

Comment:
💭 non-blocking: we should disable the animation on the `PopoverCardRoot` when `__experimental_asStandalone` is true. The assumption there is that they want to control the entrance within the popover implementation.

Similarly with the org switcher.

Example showing the animation happening when opening a radix popover that wouldn't be desired.


https://github.com/user-attachments/assets/06d49053-7f82-4dfa-a5da-053cf7aae92b

Reply:
I am wondering if we'd remove the z-index declaration too since you can see in the video above that it interferes with other elements on the page.

Reply:
1. z-index in cards need to go for sure
2. I also hear your point about the animation, can't they disable animations ? i thought we offered this.

Reply:
Its a CSS animation, so I don't think that respects the animation layout prop.

https://github.com/clerk/javascript/blob/069103c8fbdf25a03e0992dc5478ebeaeaf122ea/packages/clerk-js/src/ui/elements/PopoverCard.tsx#L24

https://github.com/user-attachments/assets/f31fa942-3e78-4c26-a247-60b686cfb1ad